app/src/main/java/com/example/teamyuml/decentworkmobile/Login.java

```diff
@@ -44,6 +53,10 @@ protected void onCreate(Bundle savedInstanceState) {
         loginInputPassword = (EditText) findViewById(R.id.login_input_password);
         btnlogin = (Button) findViewById(R.id.btn_login);
         btnLinkSignup = (Button) findViewById(R.id.btn_link_signup);
+        btnGoogleAuth = (Button)findViewById(R.id.btn_googleAuth);
+        GoogleSignInOptions signInOptions = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN).requestEmail().build();
+        final GoogleApiClient googleApiClinet = new GoogleApiClient.Builder(this).enableAutoManage(this,this).addApi(Auth.GOOGLE_SIGN_IN_API,signInOptions).build();
```

Comment:
NVM it may work but: "Accessing Google APIs with GoogleApiClient (deprecated)".
How to do it(Sample by Google):
https://github.com/googlesamples/google-services/blob/master/android/signin/app/src/main/java/com/google/samples/quickstart/signin/SignInActivity.java